plugins/by-name/lf/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "lf";
+  packPathName = "lf.nvim";
+  package = "lf-nvim";
+
+  description = ''
+    Lf file manager integration for Neovim
+  '';
+
+  maintainers = [ lib.maintainers.bpeetz ];
+
+  settingsExample = {
+    default_action = "drop";
+    default_actions = {
+      "<C-t>" = "tabedit";
+      "<C-x>" = "split";
+      "<C-v>" = "vsplit";
+      "<C-o>" = "tab drop";
+    };
+    winblend = 10;
+    dir = "";
+    direction = "float";
+    border = "rounded";
+    height.__raw = "vim.fn.float2nr(vim.fn.round(0.75 * vim.o.lines))";
```

I prefer using `nestedLiteralLua` instead of explicit `__raw`, but I won't nitpick it:

```suggestion
    height = lib.nixvim.nestedLiteralLua "vim.fn.float2nr(vim.fn.round(0.75 * vim.o.lines))";
```